internal/benchrunner/runners/system/scenario.go

```diff
@@ -32,44 +34,23 @@ type scenario struct {
 	Corpora             corpora                `config:"corpora" json:"corpora"`
 }
 
+// TODO: Why is this slightly different from the common fields?
```

Comment:
@aspacca You might know more here? And the one below?

Reply:
the data required for both `data_stream` and `corpora` configuration is different between system benchmarks and rally/stream ones.

specifically: rally/stream does not require `data_stream.vars` and `corpora.input_service`

since I "duplicated" both structs used to unmarshal the yaml configuration in the `rally`/`stream` packages that I added, I removed the fields that were not required.

it is fine to have this two structs only in the `common` package as well, with all the fields currently defined in the `system` package (ie: the "version" we have in this file). the packages where the fields are not required will just not access them. there's no impact on the specs from the underlying structs that are used to unmarshal the yaml.

the only caveat is that what can happen is that someone could add `data_stream.vars` or `corpora.input_service` in a configuration where they are not required and provide a value that cannot be unmarshalled. in this case we cannot simply ignore/not access the not required fields, the user has to remove them.

Reply:
Got it, I for now remove the 2 TODO mentions, we can always follow up on this.